core/storage/suggestion/gae_models.py

```diff
@@ -470,7 +470,8 @@ def get_in_review_suggestions_in_score_categories(
 
     @classmethod
     def get_in_review_translation_suggestions(
-        cls, user_id: str, language_codes: List[str]
+        cls, user_id: str, language_codes: List[str],
+            exp_ids: List[str]
```

Comment:
nit: deindent by 4

Reply:
Done.

Reply:
Oops, sorry, missed adding a commit which was in my stash history. Done this now.